sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -2539,13 +2539,13 @@ async function listEntities(
     case EntityType.QUEUE:
       const queueResponse = await serviceBusAtomManagementClient.getQueues({
         skip: skip,
```

Comment:
this can be simply `skip` while we're here

Reply:
Hmmm.